app/main.py

```diff
@@ -1 +1,39 @@
-#  write your code here
+class Animal:
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
```

Comment:
Read please this message:
https://mate.academy/chats/244801?selectedMessageId=3129519